tests/validation/tests/v3_api/test_service_discovery.py

```diff
@@ -244,6 +257,8 @@ def test_dns_record_type_external_ip():
     create_and_validate_dns_record(record, expected)
 
 
+# Windows could not support external IP for now.
```

Comment:
The external IP is supported by the DNS plugin in the cluster. Is there any reason Windows can't support this?

Reply:
https://kubernetes.io/docs/setup/windows/#networking-1:
> + ICMP packets directed to pass through a remote network (e.g. pod to external internet communication via ping) cannot be transposed and thus will not be routed back to their source
> + Since TCP/UDP packets can still be transposed, one can substitute ping <destination> with curl <destination> to be able to debug connectivity to the outside world.

I have changed the comment